### PR TITLE
Allow drag and drop of pictures onto layouts

### PR DESCRIPTION
**Pull request by nyalldawson**
E.g. drag an svg from a file explorer onto the layout to create a
new picture item containing that svg image.

Convenience++!

Peek 2019-11-28 14-15

### CODE REVIEW

**Comment by nirvn** [comment]
YAY!

File: src/app/layout/qgslayoutimagedrophandler.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ ***************************************************************************/
+
+#include "qgslayoutimagedrophandler.h"
+#include "qgslayoutdesignerinterface.h"
+#include "qgslayout.h"
+#include "qgsreadwritecontext.h"
+#include "qgsproject.h"
+#include "qgslayoutview.h"
+#include "qgslayoutitempicture.h"
+#include <QMessageBox>
```
Comment (nirvn) [inline src/app/layout/qgslayoutimagedrophandler.cpp]: That include (as well as the #include "qgsreadwritecontext.h" and #include "qgsproject.h" above) are not needed, should be removed.

**Comment by NathanW2** [comment]
HECK YES

**Comment by andreasneumann** [comment]
Nice!

**Review by nirvn — APPROVED** [review]

**Comment by haubourg** [comment]
THIS IS COOL

**Comment by barja8** [comment]
It's fantastic!

**Pull request merged**